### PR TITLE
Add fixtures

### PR DESCRIPTION
**Pull request by eadpearce**
To do (delete all that do not apply):

 - [x] Changelog entry added.

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -63,4 +63,7 @@ update_db_template:
 		--dbname=cms_temporary_template
 	dropdb -U postgres cms_temporary_template
 
+dummy_data:
+	ENV_FILES='secrets-do-not-commit,dev' ./manage.py loaddata fixtures/data.json
+
 .PHONY: clean pytest flake8 manage webserver requirements install_requirements css worker check_migrations
```
Comment (sdonk) [inline makefile]: Can you add the new commands to this list?

File: README.md [hunk 2]
```diff
@@ -50,6 +50,18 @@ Secrets such as API keys and environment specific configurations are placed in `
 | make css                      | Compile scss to css |
 | make secrets                  | Create your secret env var file |
 | make worker                   | Run async cache celery worker |
+| make update_db_template       | Updates the db template with any newly added migrations |
+| make setup_db                 | Drop, then set up fresh db from template |
```
Comment (richtier) [inline README.md]: how about names like:

`make database`  
`make database_template`
`make load_fixtures`

Comment (eadpearce) [reply README.md]: can we do `db_template`? i hate typing lol

Comment (sdonk) [reply README.md]: you can use tab to autocomplete :P

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@
 [Full Changelog](https://github.com/uktrade/directory-cms/compare/2019.08.27_1...2019.09.18)
 
 ### Implemented enhancements
+- No ticket - Add database fixtures and update readme with usage instructions
```
Comment (sdonk) [inline CHANGELOG.md]: please fix changelog auto merge

**Pull request merged**